rules/windows/command_and_control_remote_file_copy_scripts.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/29"
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies built-in Windows script interpreters (cscript.exe or wscript.exe) being used to download an executable file from a remote destination.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Remote File Download via Script Interpreter"
+risk_score = 43
+rule_id = "1d276579-3380-4095-ad38-e596a01bc64f"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Command and Control"]
+type = "eql"
+
+query = '''
+sequence by host.id, process.entity_id with maxspan = 30s
+  [network where process.name : ("wscript.exe", "cscript.exe") and network.protocol != "dns" and
+   network.direction == "outgoing" and network.type == "ipv4" and destination.ip !="127.0.0.1"
+  ]
+  [file where event.type == "creation" and file.extension : ("exe", "dll")]
```

Comment:
nit:

```suggestion
sequence by host.id, process.entity_id with maxspan=30s
  [network where process.name : ("wscript.exe", "cscript.exe") and network.protocol != "dns" and
   network.direction == "outgoing" and network.type == "ipv4" and destination.ip != "127.0.0.1"
  ]
  [file where event.type == "creation" and file.extension : ("exe", "dll")]
```

Reply:
May not need the `maxspan`, especially for long running scripts

Reply:
good idea